index.js

```diff
@@ -4,15 +4,271 @@ const COLLAPSE = document.querySelector('#collapse');
+    LOGO.classList.remove('collapse');
   });
 });
+
+const PROJECT_SECTION = document.querySelector('#project_section');
+
+const projects = [
+  {
+    image: './assets/images/img-placeholder.png',
+    alternative: 'Placeholder',
+    title: 'Multi-Post Stories',
+    titleSpan: ' Gain + Glory',
+    href: '#',
+    languages: ['Ruby on rails', 'CSS', 'JavaScript', 'HTML'],
+    webButton: 'See Project',
+    popupImgMobile: './assets/images/popup/mobile-img.png',
+    popupBtnClose: './assets/images/popup/close-btn.png',
+    popupTitle: 'Keeping track of hundreds of components',
+    popupText: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    popupLngMobile: ['Ruby on rails', 'CSS', 'JavaScript'],
+    popupLinks: ['See Live', 'See Source'],
+  },
+  {
+    popupImgDesktop: './assets/images/popup/desktop-img.png',
+    popupBtnCloseDesktop: './assets/images/popup/close-btn-desktop.png',
+    popupTitle: 'Keeping track of hundreds of components',
+    popupText: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    popupLngDesktop: ['Codekit', 'GitHub', 'JavaScript', 'Bootstrap', 'Terminal', 'Codepen'],
+    popupLinkLive: 'See Live',
+    popupLinkSource: 'See Source',
+  },
```

Comment:
- [ ] [REQUIRED] Since you have 6 projects in your portfolio, I humbly suggest that you also have 6 objects in the projects array with the structure below 👇🏻 

```
const projects = [
  {
    name: "Submarine game",
    description: "A Phaser3 plataformer game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
  },
  {
    ...
  },
]
```
- I think it is not realistic repeating one object for all your 6 projects, because this will give you a hard time when you need to make an update or add a new project to your portfolio 🙏🏻 

- [ ] [REQUIRED] I have also noticed that the project image that shows up on the desktop, changes on the mobile view of the same project. I kindly suggest that you maintain the same image to avoid confusion 🤔. See the image below 👇🏻 

| Desktop view | Mobile view |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/4247740/147156331-0cddf956-c0f3-4f4c-8419-703aa6b0b3a1.png) | ![image](https://user-images.githubusercontent.com/4247740/147156404-ddb2c744-dfbd-4318-a60a-f7503971bdae.png) |

- [ ] [REQUIRED] I have also noticed that the buttons in the window pop do not match the Figma design as required by the project. It would be nice if you strive to make your design as close as possible to the Figma design. Please do the same for the mobile too 😸.

| Your Design | Figma Design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/4247740/147157130-9b7b0d64-a6e9-4282-8c92-d968dea070ec.png) | ![image](https://user-images.githubusercontent.com/4247740/147157637-9adbfcaf-e738-4476-841d-edd6bb73380f.png) |